Gap3/Packages3/pcqa.html

```diff
@@ -52,7 +52,6 @@ <h3>
   Contact address
 </h3>
 <p>
-  <mixer person="Eddie Lo" data="contact"/>
+  Eddie Lo<br />email: <a href="mailto:elo@campus.hpu.edu">elo@campus.hpu.edu</a><br />
```

Comment:
Can you check if that looks right?

Reply:
That is just copied from the original webpage.

Reply:
Okay still lets check that the result is the same

Reply:
It does look the same.